pantos/servicenode/celery.py

```diff
@@ -91,3 +92,19 @@ def setup_logger(logger: logging.Logger, *args, **kwargs):
     except Exception:
         logger.critical('unable to initialize logging', exc_info=True)
         sys.exit(1)
+
+
+# Source: https://stackoverflow.com/questions/43944787/sqlalchemy-celery-with-scoped-session-error/54751019#54751019 # noqa
+@celery.signals.worker_process_init.connect
+def prep_db_pool(**kwargs):
+    """
+        When Celery fork's the parent process, the db engine & connection pool is included in that.
```

Comment:
This is not properly formatted. Also, please make sure that the line length is proper, it looks over the limit right now.

Reply:
I did copy this code over from stackoverflow, so thought on keeping as close to the original as possible :/